lib/modules/manager/gradle/parser.spec.ts

```diff
@@ -123,13 +123,21 @@ describe('modules/manager/gradle/parser', () => {
               buildTools: '30.0.3'
             ],
             kotlin: '1.4.30',
+            'spring-boot': '2.7.3',
```

Comment:
Only adding the pattern to the test input is not enough. To effectively test parsing, they should also be checked via `expect(...)`